lib/broadway/producer.ex

```diff
@@ -42,9 +42,8 @@ defmodule Broadway.Producer do
 
   You should generally modify `Broadway.Message` structs by using the functions
   in the `Broadway.Message` module. However, if you are implementing your
-  own producer, you **can manipulate** some of the struct's fields directly.
-
-  These fields are:
+  own producer, you **can manipulate** some of the struct's fields directly,
+  such as:
```

Comment:
I think the previous version reads better with the shorter sentence. Can you revert?